docs/en/operation/Scalability-Tuning.md

```diff
@@ -29,12 +29,12 @@ A third party metrics collector can be used to monitor the rate of
 change of this metric to determine how the number of files are growing over time.
 
 The number of files in Alluxio impacts the following:
-* Size of heap required by the master - Each file takes approximately 1 - 2 KB. If RocksDB is used,
-most file metadata is stored off-heap, and the size of the heap impacts how many files’ metadata can
-be cached on heap. See the
+* Size of heap required by the master - Each file and its directory structure takes approximately 4KB. 
+If RocksDB is used, most file metadata is stored off-heap, 
+and the size of the heap impacts how many files’ metadata can be cached on the heap. See the
 [RocksDB section]({{ '/en/operation/Metastore.html#rocksdb-metastore' | relativize_url }}) for more
 information.
-* Size of disk required for journal storage - Each file takes approximately 1 - 2 KB on disk.
+* Size of disk required for journal storage - Each file takes approximately 4 KB on disk.
```

Comment:
At peak, we may have 2 snapshots during checkpointing. Do we need to 2x this number for how much local disk needs to be provisioned per master?